components/Calendar/Calendar.js

```diff
@@ -0,0 +1,142 @@
+// #FFFFFF - white
+// #F5F5F5 - off-white
+// #B0B4D4 - light purp
+// #84C7D0 - sea blue
+// #4BD2D2 - ugly blue
+
+
+import { useState, useEffect } from 'react';
+// import { Scheduler, WeekView, DayView, Appointments, Toolbar, DateNavigator, ViewSwitcher,}
+//import moment from 'moment';
+import { styled, alpha } from '@mui/material/styles';
+import Paper from '@mui/material/Paper';
+import { ViewState } from '@devexpress/dx-react-scheduler';
+import {
+  Scheduler,
+  WeekView,
+  Toolbar,
+  DateNavigator,
+  Appointments,
+  TodayButton,
+  AppointmentTooltip,
+} from '@devexpress/dx-react-scheduler-material-ui';
+import TooltipContent from './subcomponents/TooltipContent.js';
+import styles from '../../utils/styles/CalendarStyles/Calendar.module.css';
+
+
+const data = [
```

Comment:
I would Recommend adding a separate exData file to dynamically call to information passed in

Reply:
Can put the example data in a temporary folder or a constants file for development